dataactvalidator/interfaces/validatorValidationInterface.py

```diff
@@ -399,6 +403,18 @@ def getRuleSeverityByName(self, ruleSeverityName):
             "Multiple rule severities found with name {}".format(ruleSeverityName))
         return query
 
+    def getLongToShortColname(self):
+        """Return a dictionary that maps schema field names to shorter, machine-friendly versions."""
+        query = self.session.query(FileColumn.name, FileColumn.name_short)
+        dict = {row.name:row.name_short for row in query}
```

Comment:
It's a dictionary comprehension, similar the list comprehensions we've been using. Basically, it's constructing a dictionary by looping through the query results.